README.md

```diff
@@ -17,19 +17,32 @@ npm i undici
 
 ## Benchmarks
 
-Machine: AMD EPYC 7502P
-
-Node 15
-```
-http - keepalive x 12,028 ops/sec ±2.60% (265 runs sampled)
-undici - pipeline x 31,321 ops/sec ±0.77% (276 runs sampled)
-undici - request x 36,612 ops/sec ±0.71% (277 runs sampled)
-undici - stream x 41,291 ops/sec ±0.90% (268 runs sampled)
-undici - dispatch x 47,319 ops/sec ±1.17% (263 runs sampled)
-```
+AMD EPYC 7502P 32 Core, Node 15
 
 The benchmark is a simple `hello world` [example](benchmarks/index.js) using a
-single unix socket with pipelining.
+number of unix sockets (connections) with a pipelining depth of 10.
+
+### Connections 1
+
+| Slower tests            | Samples | Result        | Tolerance | Difference with slowest |
+|-------------------------|---------|---------------|-----------|-------------------------|
+| http - no keepalive     |      99 |  66.66 op/sec |  ± 0.99 % |                         |
+| http - keepalive        |      99 |  69.63 op/sec |  ± 0.94 % | + 4.46 %                | 
+| undici - pipeline       |     100 | 446.97 op/sec |  ± 3.04 % | + 570.55 %              | 
+| undici - stream         |     100 | 465.31 op/sec |  ± 5.56 % | + 598.07 %              |
+| undici - request        |     100 | 486.62 op/sec |  ± 2.56 % | + 630.04 %              |
+| undici - dispatch       |     100 | 553.45 op/sec |  ± 2.67 % | + 730.29 %              |
+
+### Connections 50
+
+| Slower tests            | Samples | Result        | Tolerance | Difference with slowest |
+|-------------------------|---------|---------------|-----------|-------------------------|
+| http - no keepalive     |      99 |  31.96 op/sec |  ± 0.96 % |                         |
+| http - keepalive        |      99 |  33.74 op/sec |  ± 0.88 % | + 5.56 %                |
+| undici - pipeline       |     100 |  54.90 op/sec |  ± 1.06 % | + 71.78 %               |
+| undici - request        |     100 |  82.67 op/sec |  ± 1.69 % | + 158.65 %              |
+| undici - stream         |     100 |  94.95 op/sec |  ± 1.49 % | + 197.06 %              |
+| undici - dispatch       |     100 | 110.69 op/sec |  ± 1.57 % | + 246.33 %              |
```

Comment:
This looks odd: how come 50 connections have 1/5 op/sec than 1 connection?

There is likely something wrong in our benchmark.

Reply:
It’s how the bench are built. It’s counting “ops” however 50 connections do more per ops in order to fully saturate the connections.

Reply:
I think I can probably fix that

Reply:
Confirmed. Those are not requests but ops so indeed is misleading. 

@ronag You can probably copy the table production code from cronometro and work with raw results to apply the multiplication that was present in the old version of the benchmarks.

Or we can merge this and I'll take care of that. Let me know.

Reply:
I just made sure that the number of parallel requests are the same and then it's ok.